src/tools_old.py

```
@@ -0,0 +1,112 @@
+"""Legacy tools module for backward compatibility."""
+
+# Import the new implementations for backward compatibility
+from .services.apify.scraper_service import call_contact_details_scraper
+from .services.llm.summarization_service import summarize_contact_information
+
+__all__ = ['call_contact_details_scraper', 'summarize_contact_information']
+
+
+async def call_contact_details_scraper(
```

Ruff (0.12.2)

Redefinition of unused `call_contact_details_scraper` from line 4

(F811)